### PR TITLE
Replace Play's fakeRequest() with FakeRequestBuilder

### PR DESCRIPTION
**Pull request by jeffpw-goog**
### Description

Replace all usages of play.test.Helpers.fakeRequest() with our FakeRequestBuilder.

* Replace Helpers.fakeRequest().build() with FakeRequestBuilder.fakeRequest()
* Replace Helpers.fakeRequest() with FakeRequestBuilder.fakeRequestBuilder()
* Replace Helpers.fakeRequest(Call) with FakeRequestBuilder.fakeRequestBuilder().call(Call)
* Replace Helpers.fakeRequest(String method, String uri) with FakeRequestBuilder.fakeRequestBuilder().method(method).uri(uri)

This supports #8046.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)

### CODE REVIEW

File: server/test/controllers/admin/AdminExportControllerTest.java [hunk 1]
```diff
@@ -100,7 +101,7 @@ public void hxExportProgram_invalidProgramId_badRequest() {
     Result result =
         controller.hxExportProgram(
             addCSRFToken(
```
Comment (melanie-exygy) [inline server/test/controllers/admin/AdminExportControllerTest.java:103]: **question (non-blocking):** Do we still want to add the CSRFToken here?

File: server/test/controllers/admin/AdminProgramImageControllerTest.java [hunk 2]
```diff
@@ -61,7 +62,7 @@ public void index_ok_get() throws ProgramNotFoundException {
 
     Result result =
         controller.index(
-            addCSRFToken(fakeRequest().method("GET")).build(),
+            addCSRFToken(fakeRequestBuilder().method("GET")).build(),
```
Comment (melanie-exygy) [inline server/test/controllers/admin/AdminProgramImageControllerTest.java:65]: **question (non-blocking):** Same question here.  Aren't we adding the CSRFToken twice?

File: server/test/controllers/admin/AdminExportControllerTest.java (hunk 1 above)
Comment (jeffpw-goog) [reply server/test/controllers/admin/AdminExportControllerTest.java:103]: Coming in https://github.com/civiform/civiform/pull/8075 :) https://github.com/civiform/civiform/issues/8046 has a list of all the related PRs.

File: server/test/controllers/admin/AdminProgramImageControllerTest.java (hunk 2 above)
Comment (jeffpw-goog) [reply server/test/controllers/admin/AdminProgramImageControllerTest.java:65]: Same answer!

**Review by melanie-exygy — APPROVED** [review]

**Pull request merged**